packages/marko-web-omeda-identity-x/integration-hooks/on-login-link-sent.js

```diff
@@ -232,11 +234,14 @@ module.exports = async ({
   const idxOmedaRapidIdentify = req[idxOmedaRapidIdentifyProp];
   if (!idxOmedaRapidIdentify) throw new Error(`Unable to find the IdentityX+Omeda rapid identifier on the request using ${idxOmedaRapidIdentifyProp}`);
 
+  const promoCode = req.cookies[omedaPromoCodeCookieName] || defaultPromoCode;
+
   // get omeda customer id (via rapid identity) and load omeda custom field data
   const [{ data }, { encryptedCustomerId }] = await Promise.all([
     identityX.client.query({ query: FIELD_QUERY }),
     idxOmedaRapidIdentify({
       user: user.verified ? user : { id: user.id, email: user.email },
+      ...(promoCode && { promoCode }),
```

Comment:
The IdX rapid identify util doesn't have access to the request object (or the additional configs for the cookie name and default value). Rather than pass all that in, I created a util to determine what value to use, and added that in all 3 places. I can switch that around if you think it's warranted...let me know